### PR TITLE
feat(Cards): improve accessibility

### PR DESCRIPTION
**Pull request by marcoskolodny**
Issue: [Link](https://jira.tid.es/browse/WEB-1878)

### CODE REVIEW

File: src/card.tsx [hunk 1]
```diff
@@ -384,19 +387,14 @@ const CardContent: React.FC<CardContentProps> = ({
...
-        if (!headline) {
-            return null;
-        }
-        if (typeof headline === 'string') {
-            return <Tag type="promo">{headline}</Tag>;
-        }
-        return headline;
-    };
+
+    // title headline pretitle subtitle description extra
```
Comment (atabel) [inline src/card.tsx:396]: ??

File: src/card.tsx [hunk 2]
```diff
@@ -448,10 +459,9 @@ const CardContent: React.FC<CardContentProps> = ({
                         >
                             {description}
                         </Text2>
-                    )}
-                </Stack>
-
-                {extra && <div>{extra}</div>}
+                    </div>
+                )}
+                {extra && <Box ref={extraRef}>{extra}</Box>}
```
Comment (atabel) [inline src/card.tsx:464]: why did you change the `div` with a `Box`?

**Review by atabel — APPROVED** [review]

File: src/card.tsx (hunk 2 above)
Comment (marcoskolodny) [reply src/card.tsx:464]: Consistency. There were a few cards with Box and others with div. Do you see any reason to use div instead?

File: src/card.tsx (hunk 1 above)
Comment (marcoskolodny) [reply src/card.tsx:396]: Ups, my bad. Will remove
